docs/artifact-descriptions/sas21.md

```diff
@@ -91,7 +91,7 @@ presented in the paper quickly.
         or
 
     * Run some of the regression tests in `tests/regression/13-privatized` by calling `./regtest.sh 13 xx` where `xx` is the number of the test. Especially `xx > 16` are interesting, these were added with the paper and highlight
-      differences between different approaches. Use the `--sets exp.privatization chosenname` option to choose which thread-modular analysis to use.
+      differences between different approaches. Use the `--set exp.privatization chosenname` option to choose which thread-modular analysis to use.
```

Comment:
Do we want to update this document? Or should it be static such that it works for the artifact linked above?

Reply:
That's a good question. I think we should update the things in our repository such that they remain usable as long as possible. If one wants to use the artifact specifically, then it includes the instructions at the time. But the instructions now should reflect how to use the same stuff with the current version of Goblint (especially if we remove `--sets` completely).

This would be similar to how we handle `svcomp21.json`: if you want the exact setup from SVCOMP21, then use that verifier archive and the conf in there, if you want more-or-less equivalent configuration from the current version of Goblint, then you can use the conf from the current master.

It's not reasonable to keep these dated things locked in completely because as we fix things within Goblint, then behavior might still change. The only way to get guaranteed reproducibility is to use respective archived versions (or check out tags from git).

Reply:
I'll add a comment to this artifact description that if you're using the artifact, it includes the exact instructions there to make it clear that our documentation version needn't be identical any more.